src/api/app/controllers/api_controller.rb

```diff
@@ -68,18 +68,32 @@ def visualization_2
   def visualization_3
     @fiscal_information = School.joins(:Fiscal, :Fact)
               .select('
-                (local_revenue + state_revenue + other_revenue + fed_revenue) AS revenue,
-                school.state_lea_id,
```

Comment:
Why was this deleted? Could be useful for svg keys

Reply:
Unless Joe does some filtering on the data, you're giving him the wrong format of the data which breaks the visualization. I'm giving him exactly what he needs for his visualization.